<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomTrip</class>
 <widget class="QDialog" name="CustomTrip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom Trip</string>
  </property>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>151</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="createTrip">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Trip</string>
   </property>
  </widget>
  <widget class="QListWidget" name="addTrip">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>161</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>StartingWinery:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="startingWinery"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
